test.sh.in

```diff
@@ -218,20 +230,31 @@ setupVirtualEnv(){
 
 BASE_REPO_URL="${BASE_REPO_URL}"
 runTests() {
-  read array<<<"${TEST_GROUP}"
-  args=()
-  group=" --group="
-  for item in $array; do
-    args+="${group}${item}"
-  done
-  cp -f ${WORKSPACE}/build-config/config.ini ${WORKSPACE}/RackHD/test/config
-  cd ${WORKSPACE}/RackHD/test
-  RACKHD_BASE_REPO_URL=${BASE_REPO_URL} RACKHD_TEST_LOGLVL=INFO \
-      python run.py ${args} --with-xunit 
-  mkdir -p ${WORKSPACE}/xunit-reports
-  cp *.xml ${WORKSPACE}/xunit-reports
+  if $RUN_FIT_TEST ; then
```

Comment:
Note: presuming the jenkins problem is related to shifting this into production rather than a problem with the change, of course!